paper-tab.html

```diff
@@ -135,7 +147,12 @@
 
       _updateNoink: function() {
         this.noink = !!this.noink || !!this._parentNoink;
-      }
+      },
+
+      _focusBlurHandler: function(event) {
+        Polymer.IronControlState._focusBlurHandler.apply(this, arguments);
+        this.fire('paper-tab-' + event.type);
```

Comment:
(recap from offline discussion:) Since focus and blur don't bubble, I was firing these so there wouldn't be a need to manage focus and blur listeners of all the items. However, I hadn't considered just listening `items.*`, which should prevent the need to manually manage what nodes have what listeners, so I'll give that a shot.